contracts/BuyBackBurner.sol

```diff
@@ -143,4 +169,40 @@ contract BuyBackBurner {
 
         require(deviation <= MAX_ALLOWED_DEVIATION, "Price deviation too high");
     }
+
+    /// @dev Bridges OLAS to Ethereum mainnet for burn.
+    /// @notice if nativeTokenAmount is zero or above the balance, it will be adjusted to current native token balance.
+    /// @param nativeTokenAmount Suggested native token amount.
+    /// @param tokenGasLimit Token gas limit for bridging OLAS to L1.
+    /// @param bridgePayload Optional additional bridge payload.
+    function sendToHigherHeights(
+        uint256 nativeTokenAmount,
+        uint256 tokenGasLimit,
+        bytes memory bridgePayload
+    ) external payable {
+        require(_locked == 1, "Reentrancy guard");
+        _locked = 2;
+
+        // Buy OLAS
+        uint256 olasAmount = _buyOLAS(nativeTokenAmount);
+        require(olasAmount > 0, "Nothing to bridge");
+
+        // Burn OLAS
+        uint256 leftovers = _bridgeAndBurn(olasAmount, tokenGasLimit, bridgePayload);
+
+        // Send leftover amount, if any, back to the sender
+        if (leftovers > 0) {
+            // solhint-disable-next-line avoid-low-level-calls
+            (bool success, ) = tx.origin.call{value: leftovers}("");
+            require(success, "Leftovers transfer failed");
+        }
+
```

Comment:
Please ensure the contract can handle OLAS that is being sent to it directly. Any system should be able to send OLAS to this and then this be included in the bridge and burn logic

Reply:
That's what I thought. Great, will modify this.